### ISSUE TITLE
Loading more changesets forces user to top of page

### ISSUE THREAD

**Issue by Piszmog** [sourcegraph/sourcegraph#43122]

**Comment by chrispine** [comment]
Haven't we seen this before? Didn't we (or someone else) fix it? If so, maybe it's a quick fix, and I'd say P0. Otherwise... maybe backlog, sadly?

**Comment by courier-new** [comment]
Yeah this issue was [fixed by FPT](https://github.com/sourcegraph/sourcegraph/issues/32302), as it actually affected every place where we had a filtered connection (list) and used the URL to track the `?visible=` parameter.

I would say we should ask FPT to take a look into this, since I Laura worked on the original issue. However, from observations, it seems like the problem is isolated to the changesets list on the batch changes page. The batch changes list and site admin repositories lists, for example, don't jump when you click "show more."

Equally worrisome, though, it seems like it doesn't always jump you to the top, but I'm unclear on if there's any pattern because every time I try to record it, it does something different. In any case, this is almost certainly not a quick fix, given the precent of the original issue. 😓

**Comment by courier-new** [comment]
I think this will actually become irrelevant given we probably plan to replace the filtered connection "infinite page" lists with a 1-page-at-a-time pagination component anyway to [support managing changesets at 2XL scale](https://github.com/sourcegraph/sourcegraph/issues/16562).

**Comment by eseliger** [comment]
This might be flickering from polling 🤔 I think I saw this page hide all nodes and flash a loader briefly before

**Comment by chrispine** [comment]
Moving to Backlog since this will be fixed by the issue Kelli mentioned above
